schemas/app/InvoiceItem.json

```diff
@@ -93,16 +93,25 @@
       "label": "Stock Not Transferred",
       "fieldtype": "Float",
       "readOnly": true
+    },
+    {
+      "fieldname": "batchNumber",
```

Comment:
Why is the `batchNumber` mentioned in the Invoice?

Reply:
in case the user wants to print an Invoice with batchNumber